source/_integrations/zwave_js.markdown

```diff
@@ -20,56 +20,81 @@ ha_codeowners:
 
-### Hardware requirements
+2. [Supported Z-Wave dongle](/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules). The Z-Wave controller dongle should be connected to the same host as where the Z-Wave JS server is running. In the configuration for the Z-Wave server you need to provide the path to this stick. It's recommended to use the `/dev/serial-by-id/yourdevice` version of the path to your stick, to make sure the path doesn't change over reboots. The most common known path is `/dev/serial/by-id/usb-0658_0200-if00`.
 
-- [Supported Z-Wave dongle](/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules). The Z-Wave controller dongle should be connected to the same host as where the Z-Wave JS server is running.
+3. A **network key** used in order to connect securely to compatible devices. The network key consists of 32 hexadecimal characters, for example `2232666D100F795E5BB17F0A1BB7A146` It is recommended that a network key is configured as security enabled devices may not function correctly if they are not added securely. You must provide this network key in the configuration part of the Z-Wave JS Server. For new installations, a unique default key will be auto generated for you. TIP: You could use a site like random.org to create your own random network key. Make sure that you keep a backup of this key in a safe place  because you need to enter the same key if you decide to ever reinstall the Z-Wave JS Server.
 
-## Configuration
+4. The Z-Wave JS integration in Home Assistant. This integration connects to the Z-Wave JS Server to retrieve the info from your Z-Wave network and turns it into Home Assistant devices and entities.
 
-This integration can be configured using the integrations in the
-Home Assistant frontend.
+### Running the Z-Wave JS Server
 
-Menu: **Configuration** -> **Integrations**.
+As stated in the above requirements, you need to run the Z-Wave JS Server somewhere in your network. There are multiple ways to run this server, we'll explain the most common ways below:
 
-Click on the `+` sign to add an integration and click on **Z-Wave JS**.
-After completing the configuration flow, the Z-Wave JS integration will be
-available.
+#### 1. The official Z-Wave JS add-on, available from the add-on store
 
-### Secure network key
+This is the recommended approach if you're running the default version of Home Assistant, including the Supervisor and you do not require any advanced configuration/options but just want your Z-Wave devices available to control within Home Assistant.
 
-The secure network key is set in the settings for zwave-js-server and
-not in the integration configuration.
+#### 2. The Z-Wave JS to MQTT add-on installed available from the community add-on store
 
-## Current Limitations
+This is the recommended approach if you're running the default version of Home Assistant, including the Supervisor and you like to have maximum control over your Z-Wave network and its nodes.
```

Comment:
```suggestion
This option is only available to Home Assistant OS (the recommended installation type) and Home Assistant Supervised installations.

The Z-Wave network can be configured via the built-in Z-Wave JS control panel in Home Assistant and via the Z-Wave JS control panel built into Z-Wave JS 2 MQTT.
```

Reply:
Not true, Frenck created a HA addon for this too...